Source/FileLiberator/DownloadDecryptBook.cs

```diff
@@ -278,5 +278,34 @@ private static bool moveFilesToBooksDir(LibraryBook libraryBook, List<FilePathCa
 
 			return true;
 		}
+
+        private void DownloadCoverArt(LibraryBook libraryBook)
+		{
+            var destinationDir = AudibleFileStorage.Audio.GetDestinationDirectory(libraryBook);
+            var coverPath = AudibleFileStorage.Audio.GetBooksDirectoryFilename(libraryBook, ".jpg");
+            coverPath = Path.Combine(destinationDir, Path.GetFileName(coverPath));
+
+            try
+            {
+                if (File.Exists(coverPath)) 
+                    FileUtility.SaferDelete(coverPath);
+
+                (string picId, PictureSize size) = libraryBook.Book.PictureLarge is null ?
+                    (libraryBook.Book.PictureId, PictureSize.Native) :
+                    (libraryBook.Book.PictureLarge, PictureSize.Native);
+
+                var picBytes = PictureStorage.GetPictureSynchronously(new PictureDefinition(picId, size));
+
+                if (picBytes.Length > 0)
+                    File.WriteAllBytes(coverPath, picBytes);
+            }
+            catch (Exception ex)
+            {
+                //Failure to download cover art should not be
+                //considered a failure to download the book
+                Serilog.Log.Logger.Error(ex.Message);
```

Comment:
I generally recommend using an Error() overload that takes the exception as a param. It'll yield more helpful logs